standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/ReplChangeManager.java

```diff
@@ -467,4 +470,27 @@ static void scheduleCMClearer(Configuration conf) {
           0, MetastoreConf.getTimeVar(conf, ConfVars.REPLCMINTERVAL, TimeUnit.SECONDS), TimeUnit.SECONDS);
     }
   }
+
+  public static boolean isReplPolicySet(Database db) {
```

Comment:
Can we move this isReplPolicySet method as public method in Database object itself? That may be clean.